rabbitmq/meta/heka.yml

```diff
@@ -1,15 +1,53 @@
-{%- if pillar.rabbitmq.server is defined %}
-input:
-  logstreamer_rabbitmq:
-	type: "LogstreamerInput"
-	log_directory: "/var/log/rabbitmq"
-	file_match: 'rabbit@.+\.log\.?(?P<Index>\d+)?(.gz)?'
-	priority: ["^Index"]
-	decoder: "rabbitmq"
-	oldest_duration: "168h"
-decoder:
-  rabbitmq:
-    type: "SandboxDecoder"
-    filename: "lua_modules/decoders/rabbitmq.lua"
-    module_directory: "/usr/share/heka/lua_modules;/usr/share/heka/lua_modules/common"
-{% endif %}
+log_collector:
+  decoder:
+    rabbitmq_service:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/decoders/rabbitmq.lua
+      module_dir: /usr/share/lma_collector_modules;/usr/share/heka/lua_modules
+      config:
+        include_full_notification: false
+  input:
+    rabbitmq_log_stream:
+      engine: logstreamer
+      log_directory: "/var/log/rabbitmq"
+      file_match: 'rabbit@(?P<Node>.+)\.log$'
+      differentiator: ["rabbitmq.", "Node"]
+      decoder: "rabbitmq_decoder"
+      splitter: "rabbitmq_splitter"
+  splitter:
+    rabbitmq_splitter:
+      engine: token
+      delimiter: "\n\n(=[^=]+====)"
+      delimiter_eol: false
+metric_collector:
+  filter:
```

Comment:
@elemoine @SwannCroiset The support metadata reside inside the salt formulas and get executed within the salt runtime with variables present as are when configuring the service and have access to any system call's or salt module run's outputs. This gives you limitless power to generate whatever you need to you config. Having the default alarms definitions in the same format as other filters removes complexity.
Yes all of the defined models will be  overridable from user defined metadata, the power resides that you have well engineered checks and logic out of box and you can tune it up per need.